mitmproxy/options.py

```diff
@@ -105,13 +101,6 @@ def __init__(self, **kwargs) -> None:
             "Enable/disable HTTP/2 support. "
             "HTTP/2 support is enabled by default.",
         )
-        self.add_option(
-            "http2_priority", bool, False,
-            """
-            PRIORITY forwarding for HTTP/2 connections. Disabled by default to ensure compatibility
-            with misbehaving servers.
-            """
-        )
```

Comment:
Is the current behaviour documented? I don't mind removing it - just want to make sure we publicly state somewhere what happens with PRIORITY frames and how mitmproxy behaves when (not-)forwarding them.

Reply:
Fixed 👍